jgrapht-core/src/test/java/org/jgrapht/alg/KDisjointShortestPathsTest.java

```diff
@@ -0,0 +1,256 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2016, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* -------------------------
+ * .java
+ * -------------------------
+ * (C) Copyright 2007-2016, by France Telecom
+ *
+ * Original Author: Assaf Mizrachi and Contributors.
+ * Contributor(s):
+ *
+ * $Id$
+ *
+ * Changes
+ * -------
+ * 11-Sep-2016 : Initial revision (AM);
+ *
+ */
+package org.jgrapht.alg;
+
+import java.util.List;
+
+import org.jgrapht.EnhancedTestCase;
+import org.jgrapht.GraphPath;
+import org.jgrapht.graph.DefaultDirectedGraph;
+import org.jgrapht.graph.DefaultWeightedEdge;
+
+/**
+ * 
+ * Tests for the {@link KDisjointShortestPaths} class.
+ * 
+ * @author Assaf Mizrachi
+ */
+public class KDisjointShortestPathsTest extends EnhancedTestCase {
+
+	    
+    /**
+     * Tests single path
+     * 
+     * Edges expected in path
+     * ---------------
+     * 1 --> 2

```

Comment:
Please escape the greater than sign using the {@literal} tag. For example: {@literal1 --> 2}